### ISSUE TITLE
Autotype switched window but did not paste either username or password

### ISSUE THREAD

**Issue by Skyb0rn** [keepassxreboot/keepassxc#2349]
## Expected Behavior
Switch window, paste username, {tab}, paste password

## Current Behavior
switched window, then nothing

## Steps to Reproduce (for bugs)
I'm using MacOS Mojave, and KeePassXC 2.3.4

**Comment by droidmonkey** [comment]
Did you have the username field focused before the window switch?

**Comment by Skyb0rn** [comment]
Yes I did, and it worked before I upgraded to the latest version of XC.. (altho I forgot which version exactly... 

basically I upgraded to Mojave, still worked after with the old version of XC, and upgraded to latest version of XC and it stopped working. 

I can copy username and password with COMMAND+B and COMMAND+C. but when I use COMMAND+V, it just switches to window but does nothing..

**Comment by droidmonkey** [comment]
That could indicate a new (or changed) API under Mojave.

**Comment by Skyb0rn** [comment]
I see, is there a way I can get a version built on Xcode 9? I did not compile it myself but rather just downloaded the .dmg installer on the website

**Comment by droidmonkey** [comment]
@weslly can you assist?

**Comment by weslly** [comment]
I had the same issue with Mojave, but it works after enabling Accessibility access to KeepassXC:

![image](https://user-images.githubusercontent.com/107672/46583229-a1b0f980-ca29-11e8-8483-1d4c848d7e93.png)

**Comment by michaellass** [comment]
Now that's interesting. Let me describe in more detail how this issue showed up for me. I am using a version of KeePassXC compiled using macports.

1. I updated to Mojave. When using the autotype functionality for the first time, macOS asked me if I want to allow this and opened the settings screen @weslly attached. Afterwards, autotype worked fine.
2. I recompiled KeePassXC. Afterwards the issue as described here turned up: The active window changed but no typing took place.
3. Now I removed the entry for KeePassXC in the allowed-to-do-accessibility-stuff list.
4. On next attempt, macOS asked me again if I wanted to allow this kind of access to KeePassXC. After adding it again to the list, autotype works again.

So it seems like something changed during recompilation such that the permissions did not actually work anymore. Resetting them seems to have fixed it.
